src/components/quickstart/Windows.astro

```diff
@@ -31,7 +31,13 @@ const { latestVersion } = Astro.props
 
   <p>You’ll probably need to reboot.</p>
 
-  <p>After reboot, install an Ubuntu distro:
+  <p>After reboot, update your WSL:</p>
```

Comment:
I'm conflicted on this, because of course if you just installed and rebooted, you don't need to update. OTOH we're trying to keep this as simple and straightforward as possible.

Reply:
Then I think we don't need to add this, assuming that the info on this page is for folks who are completely new to WSL. 🙂